.devcontainer/Dockerfile

```
@@ -0,0 +1,117 @@
+    gnupg \
+    software-properties-common 	\
+    lsb-release \
+    libudev-dev \
+    zip unzip \
+    nodejs npm \
+    binutils && \
+    apt-get clean
+
+USER $USER
+
+RUN set -e -o xtrace \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)